docs/nft-example.md

```diff
@@ -26,11 +26,13 @@ You have to pull the `CAP` submodule content as follows:
 
 ```sh
 yarn cap:init
+
+npm run cap:init
```

Comment:
The documentation already mentions that'll be using "yarn" as an example and the user can use npm or any other package manager if much prefer. So, to avoid having to show every single command for any other package manager, we accept that yarn is the base of the example, so no need to show the npm example.

```suggestion
```